kura/distrib/src/main/resources/Win32-nn/start_kura_debug.bat

```diff
@@ -0,0 +1,32 @@
+@echo off
+
+set KURA_ROOT=c:\opt\eclipse
+set KURA_HOME=%KURA_ROOT%\kura\kura
+set KURA_HOME_URI=/%KURA_HOME:\=/%
+set KURA_DATA=%KURA_ROOT%\kura\data
+set KURA_TEMP=\tmp\.kura
```

Comment:
See: "if not exist c:\tmp mkdir c:\tmp"
This batch file is also intended for debug purposes only.